lib/Model.js

```diff
@@ -2,10 +2,8 @@ var DataStore = require('./DataStore');
 var stor = DataStore.store;
 
 function Model(schema) {
-  //grab schema keys
   this.schema = schema;
   this.id = null;
-  //add keys from schema if they don't exist
```

Comment:
these comments that are descriptive or noteworthy and are not commented out code are acceptable.
they may be left in.

if they are removed, then the code is written using great variable names and self-documenting.